OpenRA.Mods.Common/LoadScreens/LogoStripeLoadScreen.cs

```diff
@@ -25,6 +25,7 @@ public sealed class LogoStripeLoadScreen : BlankLoadScreen
 		float2 logoPos;
 		Sheet sheet;
 		Sprite stripe, logo;
+		string defaultMessage = "Loading...";
```

Comment:
What I mean is `string[] defaultMessage = { "Loading..."; };` would make more sense.